sidecar/cook/sidecar/__main__.py

```diff
@@ -25,32 +25,42 @@
 import os
 import sys
 
-from cook.file_server import FileServerApplication
-from cook.version import VERSION
+from cook.sidecar import file_server
+from cook.sidecar import progress
+from cook.sidecar.version import __version__
 
 
 def main(args=None):
+    log_level = os.environ.get('EXECUTOR_LOG_LEVEL', 'INFO')
+    logging.basicConfig(level = log_level,
+                        stream = sys.stderr,
+                        format='%(asctime)s %(levelname)s %(message)s')
+
     if args is None:
         args = sys.argv[1:]
```

Comment:
I brought up the fact that we should be using `argparse` on my review of the original fileserver PR... :man_facepalming:

Since I didn't force Nikolai to add it then, I'll have to do it now. :persevere:

Reply:
No good deed goes unpunished :'(

Reply:
:heavy_check_mark: